### PR TITLE
Add Buildkite, RTD and update schema

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/_data/portland-2022-config.yaml [hunk 1]
```diff
@@ -207,7 +207,13 @@ sponsors:
     #   brand: Starburst
     #   text: "<p>Starburst is on a mission to modernize data access &amp; analytics. Our company was founded in an unusual way; with customers and revenue from the beginning! Our growth is already ahead of some of the most successful software start-ups, and we don’t plan on slowing down.</p> <p>We believe our opportunity is huge. Every large company in the world suffers from a data silo problem. Traditional data warehouse products approach the problem with old solutions that breed inefficiency and ultimately can’t help business analysts run fast analytics on all their data.</p> <p>Starburst provides a modern solution that addresses these data silo &amp; speed of access problems. Starburst helps enterprises harness the value of open source Trino (formerly PrestoSQL), the fastest distributed query engine available today, by adding the tools and 24x7 support that meet the needs for big data access at scale. Ultimately, Starburst helps organizations run analytics anywhere to make better business decisions.</p>"
   first:
-
+  in_kind:
+    - name: buildkite
+      link: https://buildkite.com/
+      brand: Buildkite
+    - name: rtd
```
Comment (ericholscher) [inline docs/_data/portland-2022-config.yaml]:
```suggestion
    - name: readthedocs
```

**Review by ericholscher — APPROVED** [review]

File: docs/_data/portland-2022-config.yaml [hunk 2]
```diff
@@ -207,7 +207,13 @@ sponsors:
...
+      link: https://readthedocs.com/
+      brand: Read the Docs
```
Comment (ericholscher) [inline docs/_data/portland-2022-config.yaml:216]: Guessing the RTD image is going to be too wide, but not sure.

**Comment by plaindocs** [comment]
@ericholscher didn't look bad enough for me to ~care~ notice

image

**Comment by ericholscher** [comment]
Yea, looks fine 👍

**Comment by ericholscher** [comment]
@plaindocs looks like the RTD image is broken in production: https://www.writethedocs.org/conf/portland/2022/sponsors/#in-kind-sponsors

File: docs/_data/portland-2022-config.yaml (hunk 1 above)
Comment (plaindocs) [reply docs/_data/portland-2022-config.yaml]: @ericholscher so it is, turns out it was this change :japanese_ogre: 
fixed in 18af8b28

**Pull request merged**